core/src/scss/core/fonts/_fonts.scss

```diff
@@ -33,4 +33,5 @@
   ("woff", "ttf")
 ) {
   font-weight: 300;
+  font-display: swap;
```

Comment:
Remove from here as this is used for the Stanford logo. As @yvonnetangsu points out, swap can produce a "flash" as the fonts load. Let's prevent that from happening to the logo.